### PR TITLE
fix: revise interval config

### PR DESCRIPTION
**Pull request by Yisaer**

### CODE REVIEW

File: internal/conf/conf.go [hunk 1]
```diff
@@ -62,7 +62,7 @@ type SinkConf struct {
 	MaxDiskCache         int    `json:"maxDiskCache" yaml:"maxDiskCache"`
 	BufferPageSize       int    `json:"bufferPageSize" yaml:"bufferPageSize"`
 	EnableCache          bool   `json:"enableCache" yaml:"enableCache"`
-	ResendInterval       int    `json:"resendInterval" yaml:"resendInterval"`
+	ResendInterval       string `json:"resendInterval" yaml:"resendInterval"`
```
Comment (ngjaying) [inline internal/conf/conf.go:65]: Is it possible to parse into time.Duration directly?

File: deploy/chart/ekuiper/values.yaml [hunk 2]
```diff
@@ -160,7 +160,7 @@ kuiperConfig:
       # to enable the checkpoint mechanism
       qos: 0
       # The interval in millisecond to run the checkpoint mechanism.
-      checkpointInterval: 300000
+      checkpointInterval: 300s
```
Comment (ngjaying) [inline deploy/chart/ekuiper/values.yaml:163]: Comment has to be changed. Is it possible to compatible with int, which means default to millisecond if no duration unit.

File: internal/conf/conf.go (hunk 1 above)
Comment (Yisaer) [reply internal/conf/conf.go:65]: no

File: deploy/chart/ekuiper/values.yaml (hunk 2 above)
Comment (Yisaer) [reply deploy/chart/ekuiper/values.yaml:163]: Compatible with int will make it too complex as configuration.

**Review by ngjaying — APPROVED** [review]

**Pull request merged**